README.md

```diff
@@ -1,16 +1,2 @@
 # NCNF
 
-## Pitch - NCNF App
```

Comment:
Good thing you moved the content to another file, thanks!

Reply:
Yeah, it wasn't clean, you were right !